lrauv_ignition_plugins/test/test_mission_depth_vbs.cc

```diff
@@ -101,16 +86,19 @@ TEST_F(LrauvTestFixture, DepthVBS)
     }
   }
 
-  // Vehicle's final pose should be near the 10m mark
-  EXPECT_NEAR(tethysPoses.back().Pos().Z(), -10, 1e-1);
+  // FIXME(hidmic): hydrodynamic damping forces induce
```

Comment:
As I mentioned we should not fix this as it will require re-parameterization. I'm hoping adding the cross terms will make this go away.